setup.py

```diff
@@ -62,7 +62,7 @@
 
 install_requires = [
     'Flask>=0.11.1',
-    'Flask-CeleryExt>=0.2.2',
+    'Flask-CeleryExt>=0.2.2,<0.3.0',
```

Comment:
I think we could simply remove the constrain, i.e. `Flask-CeleryExt>=0.3.0`.

Both Flask-CeleryExt works with both, Celery 3 and 4, I would say that it's up to the other end, AKA overlay, to decide which version of celery to use, see https://github.com/CERNDocumentServer/cds/blob/cdslabs_qa/setup.py#L139